vscode-css/compile-vscode-css-languageserver.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+# @see https://github.com/mattn/vim-lsp-settings/pull/48
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="${SCRIPT_DIR}"
+SRC_DIR="${REPO_DIR}/vscode-css-languageserver"
+DIST_DIR="${REPO_DIR}/out"
+
+
+# ------------------------- #
+# download the source codes #
+# ------------------------- #
+
+pushd "${REPO_DIR}" || exit
+
+rm -rf \
+    "${SRC_DIR}" "${DIST_DIR}" \
+    "package.json" "package-lock.json"
+
+# or get the source via git clone
+# git clone --depth=1 https://github.com/vscode-langservers/vscode-css-languageserver "${SRC_DIR}"
+
+wget https://github.com/vscode-langservers/vscode-css-languageserver/archive/master.zip -O src.zip
+unzip src.zip
+rm -f src.zip
+mv vscode-css-languageserver-master "${SRC_DIR}"
+
+popd || exit
+
+
+# ------------ #
+# prepare deps #
+# ------------ #
+
+pushd "${SRC_DIR}" || exit
+
+npm install
+npm install --save-dev typescript
```

Comment:
--save-dev and --save are implied since a very long time. But that's just a note :)

Reply:
move to use `-D` now